docs/source/kernel-local.md

```diff
@@ -0,0 +1,31 @@
+## Local Mode
+
+The Local deployment is just for local development and is not meant to be run in real life operations.
```

Comment:
Let's replace 'is just for' with 'can be useful for' and 'real life operations' with 'production environments as subjects the gateway server to resource exhaustion'.

_The Local deployment can be useful for local development and is not meant to be run in production environments as it subjects the gateway server to resource exhaustion._